manifests/profiles/ambient-only.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  components:
+    cni:
+      enabled: true
+    ztunnel:
+      enabled: true
+    ingressGateways:
+      - name: istio-ingressgateway
+        enabled: false
+  values:
+    multicluster:
+      enableReader: false
+    gateways:
+      # Intentionally break gateways. Users should deploy with Gateway API only.
```

Comment:
Is this expectation currently documented anywhere, or is it one aspect of the new `ambient-only` profile that will be documented and detailed when it's added [here](https://istio.io/latest/docs/setup/additional-setup/config-profiles/)? cc/ @craigbox